cmd/debug.go

```diff
@@ -20,8 +20,13 @@ func newDebugCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// debug Cmd represents the debug command
 	var debugCmd = &cobra.Command{
 		Use:   "debug",
-		Short: "Run the local Appsody environment in debug mode",
-		Long:  `This starts a docker based continuous build environment for your project with debugging enabled.`,
+		Short: "Debug your Appsody project",
+		Long:  `Start a container-based continuous build environment for your Appsody project, with debugging enabled.`,
```

Comment:
I'm going to ask @SueChaplain, or @doveye to review the text of these changes.

@skoh7645 as part of this change set please can you clarify the descriptions for the following flags:
--name
--deps-volume
--docker-options
--network
--no-watcher
--publish
--publish-all
as it looks like all of these optional flags apply to the run, test and debug modes on the development container and I don't think that the current descriptions are clear enough. Specifically I found --deps-volume confusing, what if APPSODY_DEPS dir is already set?

Reply:
I tried it out and the command creates a docker volume using the name of the argument passed to `--deps-volume` and mounts it to the APPSODY_DEPS dir.